cli/src/commands/parse.rs

```diff
@@ -14,6 +14,7 @@ use crate::{config, print_user_warning};
 
 pub struct ParsedLockfile {
     pub path: PathBuf,
+    pub project_root: Option<PathBuf>,
```

Comment:
This seems overly complicated... Why not just strip the project root while creating the `ParsedLockfile` struct instead. That way `path` is always exactly what we submit as the lockfile path.